about.html

```diff
@@ -0,0 +1,151 @@
+
+    <div class="open">
+        <div class="close">
+            <a href="#"> <img src="img/Icon.png" alt="x icon"></a>
+        </div>
+    </div>
+
+    <div class="text-center summitabout">
+
+        <h3>Creative Commons Global Summit 2015 Logo</h3>
+        <div class="container">
+            <h5>The logo of CC Global Summit 2015 was decided through the logo competition from
+                8th June to 7th July
+                <br>After Online public voting and Selection Committee's judgment process, the logo from Naresh Agrawal
+                wont
+                the prize. <br> after consultation with Mr. Agrawal we optimized the prize-winning-logo for this year
+                summit
+                as following
+            </h5>
+        </div>
+
+        <a class="logo1" href="#"><img src="img/logo.png" alt="logo creative commons"></a>
+
+    </div>
+
+
+    <h1 class="text-center fs-3 mt-5 " style="color: rgb(0, 0, 0);">Our Activities</h1>
+    <hr class='mb-5' style="color:rgb(255, 51, 0,1) ;border: 2px solid rgb(255, 51, 0,1);width: 25%; margin-left: 37%;">
+    <p class="text-center twocassions">There are two ocasions where the Container Festival
+        came to agreement and constructed more spaces for culture</p>
+
+    <div class="imgwrapper justify-content-center">
+        <div class="singleimg">
+            <div class="imgabout p-0" style="width: 100%;">
+                <img src="img/aboutimage.jpg" alt="about image 1" width="400" height="300">
+
+                <div class="redmask">
+                    <div>
+                        <h6 class='ps-4' style="color: rgb(255, 255, 255);font-weight: lighter;">
+                            3rd January 2023
+                        </h6>
+                        <p style="color: white;font-weight: bolder;font-size: 26px;">Bogota Arena</p>
+                    </div>
+                </div>
+
+            </div>
+        </div>
+
+        <div class="singleimg">
+            <div class="imgabout p-0" style="width: 100%;">
+                <img src="img/aboutimage3.jpg" alt="about image 2" width="400" height="300">
+
+                <div class="redmask">
+                    <div>
+                        <h6 class='ps-5' style="color: white;">
+                            2nd June 2023
+                        </h6>
+                        <p style="color: white;font-weight: bolder;font-size: 26px;">Medellin Arena</p>
+                    </div>
+                </div>
+
+            </div>
+        </div>
+    </div>
+
+
+    <footer class="d-flex">
+        <img src="img/cc-logo2.png" alt="logo creative commons">
+        <div class="footertext">
+            <h6>2015 Creative Commons Korea. Some Rights Reserved.</h6>
+            <h6 class="koreantext">달리 정하지 않은 웹사이트의 모든 내용은 크리에이티브 커먼즈 저작자표시 2.0 대한민국 라이선스에 따라 이용할 수 있습니다. 자세한 이용기능 내용과
+                허락조건은
+                http://creativecommons.org/licences/by/2.0/kr 를 보세요.</h6>
+        </div>
+    </footer>
+
```

Comment:
[Required]

- [ ] As the previous reviewer mentioned, and according to the original design, The partners section should appear on the mobile version (only on the mobile version), kindly work on that 💯

| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/198815299-81bfd9c6-4bd4-42dc-a32d-7d8d55c9612f.png" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/198815339-ed7a9445-f638-47b2-841b-2cc64d0a9a73.png" width="500" /> |